stdlib/public/core/SIMDVector.swift

```diff
@@ -539,6 +563,8 @@ extension SIMD where Scalar: FixedWidthInteger {
     return Self(repeating: 1)
   }
   
+#if !$Embedded
```

Comment:
I think this can use `@_unavailableInEmbedded` (which is better because the code is still parsed).